app/assets/scss/_search-page.scss

```diff
@@ -12,23 +12,20 @@
 .button-save-wrapper {
   border-top: 1px solid $border-colour;
   border-bottom: 1px solid $border-colour;
-  padding: 20px 5px;
-  min-height: 34px;
-  margin-bottom: 20px;
+  padding-top: 20px;
+  margin: 0px 0px 20px;
 
   .save-summary-text {
-    display: inline-block;
-    font-size: 16px;
-    padding: 5px;
+    @include core-16($line-height: (36/16));
```

Comment:
This line height looks super weird with a smaller resolution.

<img width="1494" alt="screen shot 2017-10-04 at 13 36 15" src="https://user-images.githubusercontent.com/2920760/31175962-0d83f40c-a909-11e7-83cf-044c4082e977.png">

Reply:
This is caused by the `column-three-quarters` and `column-one-quarter` classes. Not sure how much we can do about that. Do you have any suggestions?

Reply:
I can add additional CSS for that size screen?

Reply:
I have added in padding instead.